Sentry_Linux.slnf

```diff
@@ -0,0 +1,49 @@
+{
+  "solution": {
+    "path": "Sentry.sln",
+    "projects": [
+      "benchmarks\\Sentry.Benchmarks\\Sentry.Benchmarks.csproj",
```

Comment:
We could generate these files with a script by crawling **/*.csproj with some filters? Then we can just define which csproj's to escape for each platform, and in CI we build a fresh new filter for each run

Reply:
do we change/add/remove project regularly enough to justify wiring code to do this?

Reply:
I wasn't thinking about how often we change, simply the fact it will be wrong eventually. Any new manual step we add should be considered carefully as it can lead to a lot of thing 'to keep in mind'.

It's just a shame these filter files are so limited that it would require scripting from our end. Feel free to merge once the duplicate script thing is removed (you merged the suggestions but ti's missing going back to a single .sh file

Reply:
Once there's a way, we can change the filters: https://github.com/getsentry/sentry-dotnet/issues/1405